### PR TITLE
Put haproxy in front of the composition

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 *~
 *.pyc
 *.pyo
+*.iml
+/.idea
```
Comment (sbrunner) [inline .gitignore]: That's should be in your ~/.gitignore

File: .gitignore [hunk 2]
```diff
@@ -13,7 +15,10 @@ __pycache__/
 /tilegeneration/config.yaml
 /print/print-apps/demo/config.yaml
 /mapserver/mapserver.map.tmpl
+/mapserver/osm.map
+/mapserver/private.map
 /mapserver/*.map.tmpl
+/mapserver/*.map.tmplq
```
Comment (sbrunner) [inline .gitignore]: tmplq?

Comment (pvalsecc) [reply .gitignore]: Yep, I don't know what generated that, though. But I do have that file.

File: .gitignore (hunk 1 above)
Comment (pvalsecc) [reply .gitignore]: OK

File: .gitignore (hunk 2 above)
Comment (sbrunner) [reply .gitignore]: The file mapserver/edit.map.tmplq.mako should be named as mapserver/edit.map.tmpl.mako :-/

**Pull request merged**